tests/covariance_matrix_06.output

```diff
@@ -0,0 +1 @@
+(12,0)
```

Comment:
But this is not matching what you say in the comment at the top of the .cc file.